src/modules/SIGNALPROCESSING/FIELDBUS_PERCENT_TO_WORD_fct.cpp

```diff
@@ -9,73 +9,68 @@
  *** This file was generated using the 4DIAC FORTE Export Filter V1.0.x NG!
  ***
  *** Name: FIELDBUS_PERCENT_TO_WORD
- *** Description:
+ *** Description: Convert a WORD Value Range 0-FAFF to a REAL in the Range 0.0 to 100.0
  *** Version:
- ***     1.0: 2023-10-21/franz -  -
+ ***     1.0: 2023-10-21/Franz Höpfinger - HR Agrartechnik GmbH -
+ ***     1.1: 2024-09-19/Franz Höpfinger - HR Agrartechnik GmbH - Update to a more Function like Interface
  *************************************************************************/
 
 #include "FIELDBUS_PERCENT_TO_WORD_fct.h"
 #ifdef FORTE_ENABLE_GENERATED_SOURCE_CPP
-#include "FIELDBUS_PERCENT_TO_WORD_fct_gen.cpp"
+#include "signalprocessing/FIELDBUS_PERCENT_TO_WORD_fct_gen.cpp"
 #endif
 
 #include "criticalregion.h"
 #include "resource.h"
-#include "forte_bool.h"
 #include "forte_real.h"
-#include "forte_word.h"
 #include "forte_udint.h"
+#include "forte_word.h"
 #include "iec61131_functions.h"
 #include "forte_array_common.h"
 #include "forte_array.h"
 #include "forte_array_fixed.h"
 #include "forte_array_variable.h"
-#include "FIELDBUS_SIGNAL_gcf.h"
+#include "signalprocessing/FIELDBUS_PERCENT_TO_WORD_fct.h"
```

Comment:
this will fail under Linux as you called the folder `SIGNALPROCESSING`

Reply:
yes, and it does fail, you are true.  (as always)
i renamed the Folder to signalprocessing to be less crying, and more consistent with the Rest of FORTE.